### PR TITLE
Quote selection

### PR DESCRIPTION
**Pull request by appinteractive**
addded quote selection

### CODE REVIEW

File: components/editor/marks/QuoteMark.js [hunk 1]
```diff
@@ -13,7 +13,10 @@ export default class QuoteMark extends Mark {
           default: 'mark-quote'
         },
         'data-file': {
-          default: 'selecting...'
+          default: 'ERROR - data-file missing'
+        },
+        'data-label': {
+          default: 'Das sollte der Name haben'
```
Comment (tillprochaska) [inline components/editor/marks/QuoteMark.js]: Huch?

File: components/editor/TactileEditor.vue [hunk 2]
```diff
@@ -98,10 +98,13 @@ export default {
   }
 }
 .editor p.is-empty:first-child::before {
-  content: 'Füge hier deinen Text ein…';
-  position: absolute;
+  content: 'Füge hier dein Manuskript per Copy&Paste ein. Dann kannst Du es mit Geräuschen und Originaltönen ausstatten.';
```
Comment (FrauCsu) [inline components/editor/TactileEditor.vue]:
```suggestion
  content: 'Kopiere den gewünschten Text hierher. Wenn du möchtest, kannst du ihn um Geräusche und Originaltöne ergänzen.';
```

File: components/editor/marks/QuoteMark.js (hunk 1 above)
Comment (roschaefer) [inline components/editor/marks/QuoteMark.js]:
```suggestion
          default: 'Hier müsste ein Dateiname stehen'
```

File: components/editor/marks/SoundMark.js [hunk 3]
```diff
@@ -14,6 +14,9 @@ export default class SoundMark extends Mark {
         },
         'data-file': {
           default: 'selecting...'
+        },
+        'data-label': {
+          default: 'Das sollte der Name haben'
```
Comment (roschaefer) [inline components/editor/marks/SoundMark.js]:
```suggestion
          default: 'Hier müsste ein Dateiname stehen'
```

File: components/editor/marks/VoiceMark.js [hunk 4]
```diff
@@ -14,6 +14,9 @@ export default class VoiceMark extends Mark {
         },
         'data-file': {
           default: 'selecting...'
+        },
+        'data-label': {
+          default: 'Das sollte der Name haben'
```
Comment (roschaefer) [inline components/editor/marks/VoiceMark.js]:
```suggestion
          default: 'Hier müsste ein Dateiname stehen'
```

**Pull request merged**